menas/ui/css/style.css

```diff
@@ -72,18 +72,27 @@ html, body {
 
 .auditTrailNestedList > ul > .sapMListNoData {
   background-color: inherit !important;
-  border-bottom: 0 !important;
-  padding-left: 0 !important;
+  border-bottom: 0px !important;
+  padding-left: 0px !important;
 }
 
 .lineageIframe {
   border: none !important;
   height: 100% !important;
 }
 
-.lineageErrorLabel {
-  font-size: 200%;
-  padding-top: 2rem;
+.scheduleMenuButton {
+  margin-top: -1rem !important;
+  position: absolute !important;
+}
+
+.scheduleMenuHBox {
+  height: 2rem !important;
+  margin-top: 1rem !important;
+}
+
+h5[id$=datasetDetailView--scheduleTimingTitle] {
```

Comment:
Possible, but quite ugly. Couldn't the element just have a custom CSS class instead of relying on the generated id?